Scripts/RemediationScripts/Remediate-EnableHTTPSForAppServices.ps1

```diff
@@ -201,9 +207,8 @@ function Enable-HttpsForAppServices
 
     if ([String]::IsNullOrWhiteSpace($context))
     {
-        Write-Host "Connecting to Azure account..."
-        Connect-AzAccount -Subscription $SubscriptionId -ErrorAction Stop | Out-Null
-        Write-Host "Connected to Azure account." -ForegroundColor $([Constants]::MessageType.Update)
+        Write-Host "AzAccount is not connected. Exiting..." -ForegroundColor $([Constants]::MessageType.Error)
```

Comment:
No active Azure login session found. Exiting...

See if this looks better

Reply:
Makes sense. 
Updated.